apps/src/lab2/constants.ts

```diff
@@ -2,18 +2,7 @@ import {AppName} from './types';
 
 export const SOURCE_FILE = 'main.json';
 
-export const BLOCKLY_LABS: AppName[] = [
-  'dance',
-  'flappy',
-  'music',
-  'thebadguys',
-  'turtle',
-  'craft',
-  'studio',
-  'bounce',
-  'poetry',
-  'spritelab',
-];
+export const BLOCKLY_LABS: AppName[] = ['dance', 'music'];
```

Comment:
Because of the above change, it means that the `AppName` type must actually be one of the keys of the `lab2Entrypoints` object, which is useful everywhere else, but also means that lists like these must be constrained to only lab2 apps.